.github/workflows/Release.yml

```diff
@@ -1,35 +0,0 @@
-name: release wasm
```

Comment:
why remove the workflows?

Reply:
I figured we'd want to use the ones we set up in cw-dex.

Reply:
Also since it's now a workspace they won't work so would have to rewrite them anyway, but then better to do as separate PR.